### ISSUE TITLE
Usage examples are no longer available

### ISSUE THREAD

**Issue by Martin-Molinero** [stanfordmlgroup/ngboost#355]
Hey!

- `http://lib.stat.cmu.edu/datasets/boston` is no longer valid, returns 404
- for `from sklearn.datasets import load_boston` we get `load_boston has been removed from scikit-learn since version 1.2.`

**Comment by ryan-wolbeck** [comment]
yes, if you take a look at tests/test_distns.py you can see how I've fixed this in the tests. Feel free to submit a PR or I'll get to it eventually

**Comment by KeerthiNingegowda** [comment]
Also, Boston dataset has a serious ethical issue. Please see  https://medium.com/@docintangible/racist-data-destruction-113e3eff54a8 . Would be great to naturally discourage the use of this dataset
